opcodes/arc-tbl.h

```diff
@@ -8043,7 +8046,10 @@
 { "jl", 0x20E20F80, 0xF8FF0FE0, ARC_OPCODE_ARCv2EM | ARC_OPCODE_ARCv2HS, JUMP, NONE, { LIMM }, { C_CC }},
 
 /* jli_s u10 010110uuuuuuuuuu.  */
-{ "jli_s", 0x00005800, 0x0000FC00, ARC_OPCODE_ARCv2EM | ARC_OPCODE_ARCv2HS, JUMP, CD1, { UIMM10_6_S }, { 0 }},
+{ "jli_s", 0x00005800, 0x0000FC00, ARC_OPCODE_ARCv2EM | ARC_OPCODE_ARCv2HS, JUMP, ARC_CD, { UIMM10_6_S }, { 0 }},
```

Comment:
For some reason when I was compiling the code after merging it into arc-2016.09 it would no longer accept the opcode with CD1 despite compiling with -mcpu=em4 -mcode-density. This was working for arc-2016.03 so I am not sure why it stopped working.